TeamCode/src/main/java/org/firstinspires/ftc/teamcode/api/Robot.java

```diff
@@ -2,73 +2,150 @@
+    public SmartServo armLeft, armRight, clawLeft, clawRight;
+    public SmartColorSensor parkSensor;
+    public WebcamName webcam0;
+    public Gyroscope gyro0, gyro1;
+    private OpMode opMode;
 
     public Robot(OpMode opMode) {
         super(opMode);
     }
 
+    // Wheels
     @Override
-    public void initTeleOp(@NonNull OpMode opMode) {
-        this.frontLeft = opMode.hardwareMap.get(DcMotor.class, Hardware.FRONT_LEFT);
-        this.frontRight = opMode.hardwareMap.get(DcMotor.class, Hardware.FRONT_RIGHT);
-        this.backLeft = opMode.hardwareMap.get(DcMotor.class, Hardware.BACK_LEFT);
-        this.backRight = opMode.hardwareMap.get(DcMotor.class, Hardware.BACK_RIGHT);
-        this.duck = opMode.hardwareMap.get(DcMotor.class, Hardware.DUCK);
+    public void powerWheels(double flPower, double frPower, double blPower, double brPower) {
+        this.fl.setPower(flPower);
+        this.fr.setPower(frPower);
+        this.bl.setPower(blPower);
+        this.br.setPower(brPower);
     }
 
+    // Arm
+
+    /*
+    * The actual arm base uses asynchronous code, so that
+    * will not be implemented until this gets tested
+    */
+
     @Override
-    public void moveAll(double power) {
-        this.frontLeft.setPower(power);
-        this.frontRight.setPower(power);
-        this.backLeft.setPower(power);
-        this.backRight.setPower(power);
+    public void positionClaw(double position) {
+        this.clawLeft.setPosition(position);
+        this.clawRight.setPosition(position);
     }
 
     @Override
-    public void stopAll() {
-        this.frontLeft.setPower(0);
-        this.frontRight.setPower(0);
-        this.backLeft.setPower(0);
-        this.backRight.setPower(0);
-        this.duck.setPower(0);
+    public void adjustClaw(double amount) {
+        // Version 1
+        this.clawLeft.setPosition(this.clawLeft.getPosition() + amount);
+        this.clawRight.setPosition(this.clawRight.getPosition() + amount);
+
+        // Version 2
+
+        /*
+        * double finalPosition = this.clawLeft.getPosition() + amount;
+        * this.clawLeft.setPosition(finalPosition);
+        * this.clawRight.setPosition(finalPosition);
+        */
     }
 
     @Override
-    public void moveFront(double power) {
-        this.frontLeft.setPower(power);
-        this.frontRight.setPower(power);
+    public void openClaw() {
+        this.positionClaw(0.9);
     }
 
     @Override
-    public void moveBack(double power) {
-        this.backLeft.setPower(power);
-        this.backRight.setPower(power);
+    public void closeClaw() {
+        this.positionClaw(0.2);
     }
 
-    public void spinDuck(double power) {
+    @Override
+    public void powerDuck(double power) {
         this.duck.setPower(power);
     }
 
-    public void setMotorsDirection(@NonNull DcMotor[] motors, DcMotor.Direction direction) {
-        for (DcMotor motor: motors) {
-            motor.setDirection(direction);
+    @Override
```

Comment:
Merge with InitRobot class

Reply:
InitRobot will not work with a class-based robot configuration. Sense we are starting to have multiple versions of the robot, the new one might have sensors and motors that the old doesn't. If we initialize this hardware in a static class that cannot change, there will be errors. Nesting the initialization inside of the Robot class means that each robot can have its own hardware. Therefor, InitRobot is going to be deleted.